### DISCUSSION TITLE
科幻電影逐漸成真，俄烏戰爭使各國更擔憂 AI 武器

### DISCUSSION THREAD

**Discussion by luo910227** [1101-GS3073-C/Python-Community-GS3073-C#391] [科技新知]
「從 2010 年代開始，聯合國就針對致命性自動武器（Lethal Autonomous Weapons，LAWs）的規範進行討論，但由於各國對於 LAWs 的定義分歧，直到 2018 年才形成初步共識，規定使用 LAWs 進行殺傷行動，必須經過人類下達最後指令，但卻又將「防衛性質」的 LAWs 排除在外。」
LAWS 致命自主武器系統(目前國際初步的共識是，若一種武器系統可「自主」選擇與攻擊目標，便可稱為LAWS。)能否應用在戰場我個人是反對的。
我覺得如果LAWS沒有受到足夠的限制，發展到一個可以自由決定要殺誰就殺誰的程度，那如果真的殺了人，它能負責嗎？最終還是得回歸設定此機器人的人，而怎麼追究又是一個課題。那個人只是設計它阿，可能不直接參與戰爭。還好目前俄烏戰爭此武器還尚未使用，不然我認為其重大影響性不亞於核武

資料來源：https://technews.tw/2022/04/29/worries-regarding-ai-weapons-increased-after-russian-ukrainian-war-broke-out/

**Comment by YuHengChen54** [comment]
唉....還是最好不要有戰爭最好

**Comment by BingRong1217** [comment]
不希望科技的發展是拿來用在會造成毀滅的部分上

**Comment by Ben6uo** [comment]
這是科技發展的副作用 只希望所有國家都是裝一下就好了
